src/AspNet.Security.OAuth.DingTalk/DingTalkAuthenticationOptions.cs

```diff
@@ -0,0 +1,51 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Microsoft.AspNetCore.Http;
+using static AspNet.Security.OAuth.DingTalk.DingTalkAuthenticationConstants;
+
+namespace AspNet.Security.OAuth.DingTalk
+{
+    /// <summary>
+    /// Defines a set of options used by <see cref="DingTalkAuthenticationHandler"/>.
+    /// </summary>
+    public class DingTalkAuthenticationOptions : OAuthOptions
+    {
+        public DingTalkAuthenticationOptions()
+        {
+            ClaimsIssuer = DingTalkAuthenticationDefaults.Issuer;
+            CallbackPath = new PathString(DingTalkAuthenticationDefaults.CallbackPath);
+
+            AuthorizationEndpoint = DingTalkAuthenticationDefaults.AuthorizationEndpoint;
+            UserInformationEndpoint = DingTalkAuthenticationDefaults.UserInformationEndpoint;
+            TokenEndpoint = DingTalkAuthenticationDefaults.TokenEndpoint;
+
+            Scope.Add("snsapi_login");
+
+            ClaimActions.MapJsonKey(ClaimTypes.Name, "nick");//用户在钉钉上面的昵称
```

Comment:
Please delete the non-Latin character comments (the maintainers don't speak the language these are written in so are not very useful).

Reply:
ok